documentation/API-onboarding.md

```diff
@@ -43,7 +43,7 @@ In parallel, the API proposal is formally registered in CAMARA by adding it to t
 
 Upon receiving the API proposal and notification from the WG chair, the TSC studies the proposal and votes it at the TSC meeting.
 
-- No-Go! Objecting companies shall provide justifications why. The API proposal is rejected and will not be included in any API Sub Project.
+- No-Go! The API proposal is rejected and will not be included in any API Sub Project, and shall either (1) be removed from the API Backlog, with objecting companies to provide justifications why, or (2) returned to the API Backlog, with the TSC including in the 'justications' section the  clarifications required/gaps to be addressed/corrections required in order for the API to be re-submitted by the API supporters.
 - Go! In this case, the TSC shall specify whether the API proposal is to be hosted by a new or existing Sub Project.
```

Comment:
Following suggestions discussed off-line, it is proposed to replace "Go!" with "Accepted".

Reply:
Changed to Accepted in latest commit